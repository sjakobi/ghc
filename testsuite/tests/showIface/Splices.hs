{-# language TemplateHaskell #-}
module Splices where

import Curries

$(genCurries 3)
