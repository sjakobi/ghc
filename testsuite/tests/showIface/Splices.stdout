[1 of 2] Compiling Curries          ( Curries.hs, Curries.o )
[2 of 2] Compiling Splices          ( Splices.hs, Splices.o )
docs:
  Just identifier env:
       module header:
         Nothing
       declaration docs:
       arg docs:
       documentation structure:
         avails:
           [curry1]
         avails:
           [curry2]
         avails:
           [curry3]
       named chunks:
       haddock options:
       language:
         Nothing
       language extensions:
         TemplateHaskell
         TemplateHaskellQuotes
       declaration locations:
         curry1:
           Splices.hs:6:3-14
         curry2:
           Splices.hs:6:3-14
         curry3:
           Splices.hs:6:3-14
       splice locations:
         Splices.hs:6:1-15

